homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/profile/header/DisplayName.kt

```diff
@@ -15,12 +16,12 @@ internal fun DisplayName(displayName: String, modifier: Modifier = Modifier) {
         text = displayName,
         style = MaterialTheme.typography.titleMedium.copy(
             fontWeight = FontWeight.Bold,
-            color = MaterialTheme.colorScheme.onSurface,
+            color = Color.White,
```

Comment:
Not sure if this is the best way to do this.

On iOS we can set a specific component to `.environment(\.colorScheme, .dark)` .
I couldn't find a simile on Jetpack Compose.

Still this should be always white, and is how the link text color is set also.

Reply:
I see! So `GravatarAppTheme(darkTheme = true)` is the trick 👍